script/configs/exclude_all_plugins_app.yaml

```diff
@@ -8,3 +8,7 @@
 
 # This is a permament entry, as it should never be a direct app dependency.
 - plugin_platform_interface
+# TODO(mvanbeusekom): Remove the exclusion of the quick_actions_ios once the 
+#                     native implementation is removed from the quick_actions 
+#                     package.
+- quick_actions_ios
```

Comment:
This should no longer be needed since it's a move now.

Reply:
Done.